torchvision/prototype/datasets/__init__.py

```diff
@@ -1,3 +1,13 @@
+try:
+    import torchdata
+except (ModuleNotFoundError, TypeError) as error:
+    raise ModuleNotFoundError(
+        "`torchvision.prototype.datasets` depends on PyTorch's `torchdata` (https://github.com/pytorch/data). "
+        "You can install it with `pip install git+https://github.com/pytorch/data.git`. "
```

Comment:
Do you need to add `@main` in the uri ? You've added it in the other commands

Reply:
Nope, if you don't specify a branch / tag / ..., `pip` uses the default branch. I'll remove it in the other places to be consistent.